packages/ttpassgen/PKGBUILD

```diff
@@ -0,0 +1,31 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=TTPassGen
+pkgver=133.a06d99d
+pkgrel=1
+pkgdesc='is a highly flexiable and scriptable password dictionary generator base on Python, you can easily use various rules to generate the desired combination of words.'
+arch=('any')
+groups=('blackarch' 'blackarch-automation' 'blackarch-cracker')
+url='https://github.com/tp7309/TTPassGen'
+license=('Apache')
+depends=('python' 'python-tqdm' 'python-click')
+makedepends=('git' 'python-setuptools')
+source=("git+https://github.com/tp7309/TTPassGen.git")
+sha512sums=('SKIP')
+
+build() {
+    cd "$pkgname"
+    python setup.py build
+}
+
+package() {
+    cd "$pkgname"
+    python setup.py install --root="$pkgdir" --prefix=/usr -O1 --skip-build
+
+    install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" README.md
+    install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
```

Comment:
Apache license is already in `/usr/share/licenses/common/Apache` so don't include it. Follow https://wiki.archlinux.org/index.php/PKGBUILD#license for licenses.

Reply:
Again. Thank you for clarification.